pylint_odoo/__init__.py

```diff
@@ -1,4 +1,7 @@
 
+__version__ = "1.3.2"
```

Comment:
FYI for the position of this variable I'm using the pep: https://www.python.org/dev/peps/pep-0008/#module-level-dunder-names